### PR TITLE
Add RPM snapshots, base one and the one for testing re-runs

### PR DESCRIPTION
**Pull request by goosemania**
re #8040
https://pulp.plan.io/issues/8040

re #8041
https://pulp.plan.io/issues/8041

### CODE REVIEW

File: rpm_base_4repos_rerun_changes/generate.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# these are the changes to the rpm_base_4repos shapshot
+pulp-admin rpm repo remove rpm --repo-id rpm-with-modules --str-eq name=bear
+pulp-admin orphan remove --all
+pulp-admin rpm repo update --repo-id rpm-with-modules --feed https://fixtures.pulpproject.org/rpm-unsigned/
+pulp-admin rpm repo update --repo-id rpm-distribution-tree --checksum-type sha1
+pulp-admin rpm repo update --repo-id srpm-unsigned --relative-url srpm
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-richnweak-deps/ --repo-id rpm-richnweak-deps --download-policy on_demand
+# need to wait for tasks to finish
+pulp-admin rpm repo sync run --repo-id rpm-richnweak-deps
+pulp-admin rpm repo remove rpm --repo-id rpm-richnweak-deps --str-eq name=icecubes
```
Comment (ipanova) [inline rpm_base_4repos_rerun_changes/generate.sh:25]: @goosemania not sure if this suggestion is not overkill and maybe out of the scope - how about copy a package from some repo to empty repo where we'd test that repo version is created but unit has not been re-migrated

Comment (goosemania) [reply rpm_base_4repos_rerun_changes/generate.sh:25]: It's a good idea, the copy part.
Let me think if I can copy it somewhere else and not to the empty repo.
I need at least one repo with no changes at all. Thanks for the suggestion!

Comment (ipanova) [reply rpm_base_4repos_rerun_changes/generate.sh:25]: just create 2 empty repos in the first snapshot? in the second snapshot you can still have one empty repo untouched, wdyt?

Comment (goosemania) [reply rpm_base_4repos_rerun_changes/generate.sh:25]: Sure, I was just thinking where to add, without redoing all snapshots.

File: rpm_base_4repos_rerun_changes/generate.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# these are the changes to the rpm_base_4repos shapshot
+pulp-admin rpm repo remove rpm --repo-id rpm-with-modules --str-eq name=bear
+pulp-admin orphan remove --all
+pulp-admin rpm repo update --repo-id rpm-with-modules --feed https://fixtures.pulpproject.org/rpm-unsigned/
+pulp-admin rpm repo update --repo-id rpm-distribution-tree --checksum-type sha1
+pulp-admin rpm repo update --repo-id srpm-unsigned --relative-url srpm
+pulp-admin rpm repo create --feed https://fixtures.pulpproject.org/rpm-richnweak-deps/ --repo-id rpm-richnweak-deps --download-policy on_demand
+pulp-admin rpm repo sync run --repo-id rpm-richnweak-deps
+pulp-admin rpm repo remove rpm --repo-id rpm-richnweak-deps --str-eq name=icecubes
+pulp-admin rpm repo copy rpm --from-repo-id rpm-richnweak-deps --to-repo-id rpm-empty-for-copy --str-eq name=Scotch
```
Comment (goosemania) [inline rpm_base_4repos_rerun_changes/generate.sh:26]: copy case

**Comment by goosemania** [comment]
@ipanova , ready for re-review, I added your suggestion and regenerated the snapshots

**Review by ipanova — APPROVED** [review]

**Pull request merged**